### PR TITLE
Remove overlapping of footer. Fixes #63, #88

### LINKED ISSUE #63: UI bug on successful '/upload'
**I'm submitting a ...**  (check one with "x")
- [X] bug report
- [ ] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/badgeyay

**Current behavior:**

by1

Footer overlaps the form whenever '/upload' is successful.
**Expected behavior:**

The footer must not overlap with the form.
It should stay either fixed on bottom or should be placed below the form everytime.

**Steps to reproduce:**

Changing some CSS would definitely do.
:D

### PR DESCRIPTION
**Pull request by Pk13055**
Fixes #

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

#### Changes proposed in this pull request:

-
-
-

### CODE REVIEW

**Comment by Pk13055** [comment]
#63, #88

File: app/templates/index.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
 				{% if m %}
                     {% for cat,msg in m %}
                         {% if cat == 'success' %}
-                            <div class="flash-success">Your badges has been successfully generated.</div>
+                            <div class="flash-{{cat}}">Your badges has been successfully generated.</div>
```
Comment (jajodiaraghav) [inline app/templates/index.html]: Why did you made this change?

File: app/templates/layout.html [hunk 2]
```diff
@@ -15,10 +15,11 @@
 	<script src="{{ url_for('static', filename='js/jquery.js') }}"></script>
   <script src="{{ url_for('static', filename='js/bootstrap.js') }}"></script>
 	<script src="{{ url_for('static', filename='js/script.js') }}"></script>
-
-  <div class="footer">
-    <strong>Version:</strong>
-    <a class="version"></a>
+  <div id="footer" class="navbar-static-bottom">
```
Comment (jajodiaraghav) [inline app/templates/layout.html]: We do not want this static

File: app/static/css/style.css [hunk 3]
```diff
@@ -147,15 +147,6 @@
 textarea {
   resize: vertical;
 }
-.footer {
-  position: absolute;
```
Comment (jajodiaraghav) [inline app/static/css/style.css]: All you need to do is change absolute to relative. that's it!

**Comment by jajodiaraghav** [comment]
Hi @Pk13055 Please fix the conflicts so that this can be merged 😄 

In case you face any difficulty or want to learn how to do a particular thing, head over to chat https://gitter.im/fossasia/badgeyay and ask us there ⭐️

**Comment by Pk13055** [comment]
@jajodiaraghav Please review.

**Comment by jajodiaraghav** [comment]
@Pk13055 You did not make any of the changes I asked you to. Also, you created 3 commits for just one simple fix.

File: app/templates/index.html [hunk 4]
```diff
@@ -16,6 +16,7 @@
 
 	                    {% elif cat == 'success-pdf' %}
 	                    	<div class="text-center">
+		                    	<div class="flash-{{cat}}">Your badges has been successfully generated.</div>
```
Comment (jajodiaraghav) [inline app/templates/index.html]: Why did you add this line?

**Comment by niccokunzmann** [comment]
@jajodiaraghav What would you like him to do? Would it be ok if @Pk13055 created an new pull-request with a single change: `position: absolute` to `position: relative`? Would you merge that?

Also
> A patch SHOULD be a minimal and accurate answer to exactly one identified and agreed problem.

https://rfc.zeromq.org/spec:42/C4/
I am not sure if this requirement is met.

@Pk13055 Would you like to create a new pull-request which is smaller?

**Comment by Pk13055** [comment]
@niccokunzmann I shall create a new PR with the requested changes.

**Comment by niccokunzmann** [comment]
@Pk13055 Yes, please.

**Comment by niccokunzmann** [comment]
I also hope that this is not too discouraging to you. I would like to make our motivations clear to you, so you see you are welcome here and policies we try to follow make sense but sometimes mean more work in the short run but keep the project going in the long run.

**Comment by Pk13055** [comment]
Yeah I understand. I actually tried to merge jishnu's dev branch into mine
and fix the conflicts but I guess the fix was pending as well. Nonetheless,
I shall push soon.
